src/azure-cli-core/azure/cli/core/util.py

```diff
@@ -89,8 +89,11 @@ def handle_exception(ex):  # pylint: disable=too-many-return-statements
             return 1
         if isinstance(ex, HttpOperationError):
             try:
-                response_dict = json.loads(ex.response.text)
-                error = response_dict['error']
+                response = json.loads(ex.response.text)
+                if isinstance(response, str):
+                    error = response
+                else:
+                    error = response['error']
```

Comment:
What if the response type is a list?

Reply:
I believe it's impossible, it's already weird enough being a str

Reply:
Is it possible 'error' is not in response?

Reply:
It's decided by service response, so I am not sure. But currently, we only found the cases that response is a tring type.